odoo/models.py

```diff
@@ -2334,7 +2334,7 @@ def _read_group_format_result(self, data, annotated_groupbys, groupby, domain):
             # full domain for this groupby spec
             d = None
             if value:
-                if ftype == 'many2one':
+                if ftype in ['many2one', 'many2many']:
```

Comment:
tuples are faster (VM optimization)
```suggestion
                if ftype in ('many2one', 'many2many'):
```